app/components/bali/reveal/index.scss

```diff
@@ -7,14 +7,17 @@
     align-items: center;
     padding-bottom: $size-4;
     margin-bottom: $size-4;
-    border-bottom: 0.0625rem solid rgba(0, 0, 0, 0.23);
-  
+
     .icon {
       height: $size-7;
       transform: rotate(270deg);
     }
   }
 
+  .is-border-bottom {
+    border-bottom: 0.0625rem solid rgba(0, 0, 0, 0.23);
```

Comment:
¿Qué opinas de poner el color y el grosor en variables para poder sobre escribirlas?

Reply:
@MiguelFrias97  Sounds good to me... Let me update..